internal/pkg/manifest/workload_ecs.go

```diff
@@ -210,12 +210,20 @@ func (s *Secret) IsSecretsManagerName() bool {
 	return !s.fromSecretsManager.IsEmpty()
 }
 
+// SSMRequiresImport returns true if the SSM parameter name or secret ARN value is imported from CloudFormation stack.
+func (s *Secret) SSMRequiresImport() bool {
```

Comment:
nit: I feel like the `SSM` prefix is unnecessary and potentially misleading (since it works for secrets manager ARN as well)

```suggestion
// RequiresImport returns true if the SSM parameter name or secret ARN value is imported from CloudFormation stack.
func (s *Secret) RequiresImport() bool {
```